### PR TITLE
When we added the method, we also should have added the marker interface

### PR DESCRIPTION
**Pull request by roesslerj**
Should solve issue https://github.com/retest/recheck-web/issues/290

### CODE REVIEW

File: src/main/java/de/retest/web/selenium/WebElementWrapper.java [hunk 1]
```diff
@@ -10,8 +10,9 @@
 import org.openqa.selenium.Rectangle;
 import org.openqa.selenium.WebDriverException;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.WrapsElement;
 
-public class WebElementWrapper implements WebElement {
+public class WebElementWrapper implements WebElement, WrapsElement {
 
 	private final WebElement delegate;
```
Comment (beatngu13) [inline src/main/java/de/retest/web/selenium/WebElementWrapper.java]: For convenience and consistency, rename to `wrappedElement`?

Comment (roesslerj) [reply src/main/java/de/retest/web/selenium/WebElementWrapper.java]: We can do that, but please in a separate PR. Small steps...

Comment (roesslerj) [reply src/main/java/de/retest/web/selenium/WebElementWrapper.java]: And this is a bugfix, that we can include in a hotfix release. Your suggestion would be a major change that we needed to test again.

Comment (beatngu13) [reply src/main/java/de/retest/web/selenium/WebElementWrapper.java]: Why would that be a "major change"? I'm talking about renaming the field, of which the scope is limited to this class. From:

```java
@Override
public WebElement getWrappedElement() {
	return delegate;
}
```

To:

```java
@Override
public WebElement getWrappedElement() {
	return wrappedElement;
}
```

The getter says `getWrappedElement`, not `getDelegate`. This just follows conventions and avoids e.g. problems with Lombok. IMHO this change belongs in this PR.

Comment (roesslerj) [reply src/main/java/de/retest/web/selenium/WebElementWrapper.java]: Ok, then I misunderstood... Done

**Pull request merged**